modules/end-user-guide/examples/snip_che-configuring-a-workspace-to-use-an-intellij-based-ide.adoc

```diff
@@ -0,0 +1,55 @@
+.Procedure
+
+. When using a JetBrains IDE requesting a registration, see xref:provisioning-jetbrains-activation-code-for-offline-use.adoc[].
```

Comment:
This (first) step does not appear to be part of this procedure both in terms of the user experience and in terms of docs structure. Consider removing this step from here completely because you already have a separate section for that whole procedure - if the user encounters that situation, the user will naturally look up that other procedure in the docs.